docs/config/README.md

```diff
@@ -651,11 +651,22 @@ The module will be shown if any of the following conditions are met:
 
 ### Options
 
-| Variable   | Default         | Description                                                     |
-| ---------- | --------------- | --------------------------------------------------------------- |
-| `symbol`   | `"💧 "`         | The symbol used before displaying the version of Elixir/Erlang. |
-| `style`    | `"bold purple"` | The style for the module.                                       |
-| `disabled` | `false`         | Disables the `elixir` module.                                   |
+| Variable   | Default                                                   | Description                                                     |
+|------------|-----------------------------------------------------------|-----------------------------------------------------------------|
+| `symbol`   | `"💧 "`                                                   | The symbol used before displaying the version of Elixir/Erlang. |
+| `style`    | `"bold purple"`                                           | The style for the module.                                       |
+| `format`   | `"via [$symbol$version \\(OTP $otp_version\\)]($style) "` | The format for the module elixir.                               |
+| `disabled` | `false`                                                   | Disables the `elixir` module.                                   |
+
+### Variables
+
+| Variable | Example | Description                          |
+|----------|---------|--------------------------------------|
+| version  | `v1.10` | The version of `elixir`              |
+| symbol   |         | Mirrors the value of option `symbol` |
```

Comment:
I think we need to add the `$otp_version` variable here.

Reply:
```suggestion
|otp_version|         | The otp version of `elixir`          |
| symbol    |         | Mirrors the value of option `symbol` |
```